src/routes/+page.svelte

```diff
@@ -53,6 +57,19 @@
         <li>Providing solutions to <em>your</em> problems.</li>
     </ul>
 </section>
+<section class="prose max-w-full">
+    <h1 class="font-dm">Why UP CSI?</h1>
+    <ContentContainer
+        topText="For organizations"
+        headerText="We offer services catered to your specific software needs."
+        bottomText="Automate the busy work in your day-to-day operations so people can get home on time more often."
```

Comment:
Let's use [named slots](https://learn.svelte.dev/tutorial/named-slots) here instead. Normally, when we deal with content, it's better to treat them as slotted elements rather than props.